starters/express-apollo-prisma/docker-compose.yaml

```diff
@@ -30,9 +30,11 @@ services:
   rabbitmq:
     image: rabbitmq:3.8-management-alpine
     container_name: 'rabbitmq'
+    restart: unless-stopped
+    env_file: ./.env
```

Comment:
I believe we can drop all the `env_file: ./.env` lines. We don't use our local variables in the containers.